### PR TITLE
updated link and added few cannels

### LINKED ISSUE #4148: Time TV
**Channel Name:** Time TV
**Country:** IN
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** https://mhdtvworld.xyz/live/time-tv/
**Notes (optional):** xxx

### LINKED ISSUE #4168: Time Vision News
**Channel Name:** Time Vision News
**Country:** IN
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** http://expressott.in/e09c06ff-fbf4-46dc-a56f-ad4a872f1900
**Notes (optional):** xxx

### LINKED ISSUE #4197: Janam TV
The link is not broken, but it's in 220p , there is a youtube stream for thee channel with 1080p.

**_Channel Name:_** Janam TV
**_Original Link (from playlist):_** https://vidcdn.vidgyor.com/janamtv-origin/liveabr/janamtv-origin/live3/playlist.m3u8
**_Replacement Link Proposed:_** https://youtu.be/_WK30gnY3_4
**_Notes (optional):_** xxx

### PR DESCRIPTION
**Pull request by Niyasmkkv**
Closes https://github.com/iptv-org/iptv/issues/4197
Closes https://github.com/iptv-org/iptv/issues/4168
Closes https://github.com/iptv-org/iptv/issues/4148

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -186,8 +186,8 @@ https://live-indiatvnews.akamaized.net/indiatv-origin/liveabr/playlist.m3u8
 http://103.199.161.254/Content/jaihind/Live/Channel(Jaihind)/index.m3u8
 #EXTINF:-1 tvg-id="JanTV.in" tvg-name="Jan TV" tvg-country="IN" tvg-language="Hindi" tvg-logo="" group-title="",Jan TV (360p)
 http://jantvstream.in:1935/edge1/sc1jantv.stream_aac/playlist.m3u8
-#EXTINF:-1 tvg-id="JanamTV.in" tvg-name="Janam TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Entertainment",Janam TV (240p)
-https://vidcdn.vidgyor.com/janamtv-origin/liveabr/janamtv-origin/live3/playlist.m3u8
```
Comment (Dum4G) [inline channels/in.m3u:190]: We don't remove an alternate sources if those keep working. A link from @LaneSh4d0w is offering 720p
https://vidcdn.vidgyor.com/janamtv-origin/liveabr/playlist.m3u8

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/in.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/in.m3u:190]: Took care of it.

Comment (Niyasmkkv) [reply channels/in.m3u:190]: This link taking time to load and missing audio after a while for me in VLC.

Comment (LaneSh4d0w) [reply channels/in.m3u:190]: It takes time because it's bumping quality. Purpose of an adaptive playlist.

File: channels/in.m3u [hunk 2]
```diff
@@ -186,8 +186,10 @@ https://live-indiatvnews.akamaized.net/indiatv-origin/liveabr/playlist.m3u8
...
+#EXTINF:-1 tvg-id="JanamTV.in" tvg-name="Janam TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Entertainment",Janam TV (720p)
```
Comment (Dum4G) [inline channels/in.m3u:189]: use tvg-logo from the link below

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**